profiles/rasterbot.lua

```diff
@@ -0,0 +1,42 @@
+-- Rasterbot profile
+
+-- Minimalist node_ and way_functions in order to test source_ and segment_functions
+
+function node_function (node, result)
+end
+
+function way_function (way, result)
+  local highway = way:get_value_by_key("highway")
+  local name = way:get_value_by_key("name")
+
+  if name then
+    result.name = name
+  end
+
+  result.forward_speed = 15
+  result.backward_speed = 15
+end
+
+function source_function ()
+  raster_source = sources:load(
+    "../test/fixtures/rastersource.asc",
```

Comment:
@emiltin possibly, but to do that this feature would become limited to one data source. When everything is done in the profiles like this you can include multiple sources, each of which are assigned their own int ID (in this case `raster_source` is `0`) and can be queried separately (so e.g. you could layer elevation data _and_ pollution data, or you could geographically patchwork multiple sources together, etc).